src/Spatial/Euclidean/Plane.cs

```diff
@@ -74,6 +77,30 @@ public Plane(Point3D rootPoint, UnitVector3D normal)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Plane"/> struct.
+        /// Constructs a Plane from the given points.
+        /// </summary>
+        /// <param name="points">The points</param>
+        /// <returns>Fitted <see cref="Plane"/></returns>
+        /// <remarks>this method uses SVD to fit the plane.</remarks>
+        public static Plane CreateFittedPlaneFrom(IEnumerable<Point3D> points)
```

Comment:
I think we need to locally transform the `IEnumerable<Point3D>` to a `List<Point3D>`. Then we wouldn't traverse the collection multiple times.

I also think that a better name would just be `BestFit`